config/sync/block.block.pagetitle.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
 id: pagetitle
```

Comment:
This is interesting to me. Normally I'd expect to see page titles moved between regions using preprocess code; feeding that into a suitable template. Is this wrapped in a block to be easier to incorporate with tools such as Layout Builder or for other reasons?

Reply:
This is a core block and while it has been placed in a region in the block layout, this is merely to ensure that a title is always rendered out when viewing a given page - as content types have gotten their layouts sorted article, application etc I have been turned this block off, so by the time we get the site fully themed we probably won't need it and it shall be removed

Reply:
Okay, good to know. Thanks!